udacity/log-analysis-project/reporting.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
```

Comment:
2 or 3 ?

Reply:
Excuse me, could you explain what do you mean?